cmd/snap/cmd_prepare_image.go

```diff
@@ -92,6 +99,19 @@ func (x *cmdPrepareImage) Execute(args []string) error {
 		Architecture: x.Architecture,
 	}
 
+	if x.Customize != "" {
+		f, err := os.Open(x.Customize)
+		if err != nil {
+			return fmt.Errorf("cannot read image customizations: %v", err)
+		}
```

Comment:
Do we need a `defer f.Close()` here or is something else closing this file?

Reply:
no, thanks for spotting, we need it, but probably the code needs a reorg, is a bit weird to keep that file open until we have created the image

Reply:
done